hathor/transaction/block.py

```diff
@@ -349,3 +349,11 @@ def verify(self, reject_locked_reward: bool = True) -> None:
         self.verify_parents()
 
         self.verify_height()
+
+    def get_feature_activation_bits(self) -> int:
+        """Returns the feature activation bits from the signal bits."""
+        assert self.signal_bits <= 0xFF, 'signal_bits must be one byte at most'
+
+        bitmask = (1 << settings.FEATURE_ACTIVATION.max_signal_bits) - 1
```

Comment:
Hmm I'm not sure about that, since the bitmask's value is specifically tied to how the `Block` interprets its own `signal_bits` field.

If we were to change the Block's interpretation, for example, changing from using the rightmost bits to the leftmost bits. That would make the bitmask change. But to me this is a change completely under the Block scope, not generic enough for Feature Activation settings scope. What do you think?

Reply:
My initial idea was that it does not depend on the block but I agree with you. Maybe move it to a `Block.get_feacture_activation_mask()` method?

Reply:
Moved in https://github.com/HathorNetwork/hathor-core/pull/613/commits/537e883413c022a020973b99b20ce2e08874460a

> My initial idea was that it does not depend on the block

What do you mean?

Reply:
I meant that the calculation of the bit mask does not depend on any block attribute.